### PR TITLE
chore(deps): update `@shelf/elasticsearch-local` to v1.0.1

### PR DESCRIPTION
**Pull request by gregholland**
Update `@shelf/elasticsearch-local` to take advantage of indexes failing to create if the test runner doesn't complete. https://github.com/shelfio/elasticsearch-local/pull/56

### CODE REVIEW

**Comment by gregholland** [comment]
Any issue if I update this PR to add the caret to the version so it will automatically pull in patch and minor version bumps?

**Comment by vladgolubev** [comment]
No problem, go ahead!

File: package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
     "preset": "./jest-preset.js"
   },
   "dependencies": {
-    "@shelf/elasticsearch-local": "1.0.0",
+    "@shelf/elasticsearch-local": "1.0.1",
```
Comment (vladgolubev) [inline package.json:52]:
```suggestion
    "@shelf/elasticsearch-local": "^1.0.1",
```

**Review by vladgolubev — APPROVED** [review]

**Comment by vladgolubev** [comment]
Published https://github.com/shelfio/jest-elasticsearch/releases/tag/v1.0.1

**Pull request merged**